package.json

```diff
@@ -56,6 +56,7 @@
     "json-schema": "^0.4.0",
     "json-schema-traverse": "^1.0.0",
     "markdown-table": "^3.0.3",
+    "prettier": "^3.4.2",
```

Comment:
I think there may be a way to avoid moving prettier to dependencies.

We can add it to `peerDependencies` instead and lazily import it for consumers who have it installed:

```ts
async function getEndOfLineFromPrettierConfig(): Promise<'\n'|'\\r\\n'|undefined> {
  let prettier: typeof import('prettier') | null = null;
  try {
    prettier = await import('prettier');
  } catch {
    return undefined; // Prettier not installed by consumer
  }
  const opts = await prettier.resolveConfig(process.cwd());
  if (!opts) return undefined;
  if (opts.endOfLine === 'lf') return '\n';
  if (opts.endOfLine === 'crlf') return '\r\n';
  return '\n';
}
```

Result: Consumers with Prettier get detection. Others don’t pull Prettier at runtime and still fall back cleanly.